src/features/ProfileView/Profile.Greeting.js

```diff
@@ -35,7 +35,7 @@ const ProfileGreeting = () => {
         />
       </View>
 
-      <Text style={styles.profileName}>Hi, I'm {accountHolderName}</Text>
+      <Text style={styles.profileName}>Welcome back {accountHolderName}</Text>
```

Comment:
lol could we use just 'Welcome' or sth else? 'Welcome back' feels not very accurate to me cuz it's not the very first screen the user sees when log back in?

Reply:
Sure thing! I was thinking that needed tweaking as well.